static/js/stream_list.js

```diff
@@ -422,6 +422,15 @@ export function update_streams_sidebar(force_rerender) {
 }
 
 export function update_dom_with_unread_counts(counts) {
+    if (!counts.size) {
```

Comment:
Why is `counts.size` the right thing to check?

I wonder if instead `counts.stream_count` should be designed to contain every stream, not just those with nonzero unreads.

Reply:
> Why is counts.size the right thing to check? 

Give that in the current state `unread_topic_counter` is empty, we should be able to safely assume there are no unread topics. If it is not being updated properly from somewhere, we should just fix it there.

Reply:
Yeah, it should be `counts.stream_count`. Fixed!